cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/cli/TabReportGenerator.java

```diff
@@ -397,8 +397,17 @@ protected String apply(SampleProvenance lims) {
                 })
             .orElse("NoIndex"));
     cs.add(""); // IUS SWIDs
-    cs.add(""); // IUS attributes
 
+    cs.add(record.workflow().getExternalKeys().stream().filter(
+            key -> {
+                var keyId =
+                        key.getVersions()
+                        .getOrDefault("pinery-hash-" + record.formatRevision(), "")
+                        .equals(record.lims().getVersion()) ? key.getId() : "";
+                return key.getId().equals(keyId); // Return keys that match external id of LIMS version key
+            })
+            .map(key -> key.getVersions().getOrDefault("shesmu-sha1", ""))
+            .collect(Collectors.joining())); // IUS Attributes
```

Comment:
Can this attribute be changed to the key=value format? (e.g. shesmu-sha1=99A57919A70676AFCDEFB84B07E116D9E2E566B3)